OpenRA.Game/Traits/World/ActorMap.cs

```diff
@@ -47,12 +47,12 @@ class CellTrigger
 			public readonly CPos[] Footprint;
 			public bool Dirty;
 
-			readonly Action<Actor> onActorEntered;
-			readonly Action<Actor> onActorExited;
+			readonly Action<Tuple<Actor, Player>> onActorEntered;
+			readonly Action<Tuple<Actor, Player>> onActorExited;
```

Comment:
I believe it would be cleaner (also somewhat more efficient) to maintain a `readonly Action<Tuple<Actor, Player, Player> changedOwner` instead of adding `Player`s to all of our existing collections.

This is what I was asking for in my earlier requests, but looking back at what I wrote that probably wasn't clear.